.github/workflows/functional.yml

```diff
@@ -61,22 +48,13 @@ jobs:
         uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 # v5.5.1
         with:
           images: |
-            name=ghcr.io/nginxinc/nginx-gateway-fabric/${{ matrix.nginx-image }}
+            name=ghcr.io/nginxinc/nginx-gateway-fabric/${{ inputs.image }}
           tags: |
             type=semver,pattern={{version}}
             type=edge
             type=ref,event=pr
             type=ref,event=branch,suffix=-rc,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
 
-      - name: Build binary
-        uses: goreleaser/goreleaser-action@7ec5c2b0c6cdda6e8bbb49444bc797dd33d74dd8 # v5.0.0
-        with:
-          version: latest
-          args: build --snapshot --clean
-        env:
```

Comment:
will it still be possible to use a separate NGF image with the configured telemetry parameters? this is needed for the functional test for telemetry